src/Microsoft.DotNet.Cli.Utils/DotnetFiles.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public static class DotnetFiles
+    {
+        /// <summary>
+        /// The CLI ships with a .version file that stores the commit information and CLI version
+        /// </summary>
+        public static string VersionFile => Path.GetFullPath(Path.Combine(typeof(DotnetFiles).GetTypeInfo().Assembly.Location, "..", "..", ".version"));
```

Comment:
Did the change get in?

Some linux CIs passed the "modified version file" test: http://dotnet-ci.cloudapp.net/job/dotnet_cli/job/rel_1.0.0/job/release_ubuntu_x64_prtest/66/testReport/Microsoft.DotNet.Tools.Builder.Tests/IncrementalTests/

It specifically asserts for the file existence (the copy that gets there from inside stage2)